src/transformStateWithClones.js

```diff
@@ -8,6 +8,43 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = { ...state };
+  const arrResult = [];
+
+  for (const action of actions) {
```

Comment:
destruct your action

```suggestion
  for (const action of actions) {
  const { type, extraData, keysToRemove } = action;
```